### PR TITLE
Version 1.1.0

### PR DESCRIPTION
**Pull request by pedropaulofb**
Description and metadata improved.
Allignment with OntoUML Metamodel v1.1.0.
Documentation using PyLODE.
New examples created for complete models.

### CODE REVIEW

**Comment by pedropaulofb** [comment]
Please keep in mind that it will be necessary to adjust the vocabulary page's links after the creation of the GitHub page (domain github.io).

File: README.md [hunk 1]
```diff
@@ -1,15 +1,36 @@
-# OntoUML Metamodel Vocabulary
+# OntoUML Vocabulary
 
-An implementation of the OntoUML metamodel in the Web Ontology Language compliant with the [ontouml-js](https://github.com/OntoUML/ontouml-js) and the [ontouml-schema](https://github.com/OntoUML/ontouml-schema). 
+<p align="center"><img src="docs/logo-vocabulary.png" width="512"></p>
```
Comment (tgoprince) [inline README.md:4]: There is no file at `docs/logo-vocabulary.png`

File: ontouml.ttl [hunk 2]
```diff
@@ -1,303 +1,800 @@
...
+
+                            owl:versionIRI <https://w3id.org/ontouml/1.1.0> ;
+                            owl:versionInfo "General review for compliance with the OntoUML Metamodel 1.1.0."@en ;
+                            owl:priorVersion <https://w3id.org/ontouml/1.0.0> ;
+                            dct:issued "2022-04-03"^^xsd:date ;
+                            dct:modified "2023-07-23"^^xsd:date ;
+
+################################################################# IDENTIFICATION
+
+                            dct:title "OntoUML Metamodel Vocabulary"@en ;
```
Comment (tgoprince) [inline ontouml.ttl]: You changed the name of the project, but did not change the name in the file. Please change all occurrences of "OntoUML Metamodel Vocabulary".

File: ontouml.ttl [hunk 3]
```diff
@@ -1,303 +1,800 @@
...
+                                         "OntoUML-Metamodel"@en ,
+                                         "OntoUML-Vocabulary"@en ,
+                                         "Ontology-Driven Conceptual Modeling"@en ;
+
+################################################################# AUTHORSHIP
+
+                            dct:publisher <https://www.utwente.nl/en/eemcs/scs/> ;
+
+                            dct:creator [ sdo:name "Tiago Prince Sales"@en ;
+                                          sdo:identifier <https://orcid.org/0000-0002-5385-5761>;
```
Comment (tgoprince) [inline ontouml.ttl]: We should use dct:identifier instead.

File: ontouml.ttl [hunk 4]
```diff
@@ -1,303 +1,800 @@
...
+                                        [ sdo:name "Pedro Paulo Favato Barcelos"@en ;
+                                          sdo:identifier <https://orcid.org/0000-0003-2736-7817>;
+                                          sdo:affiliation <https://www.utwente.nl/en/eemcs/scs/> ; ] ;
+
+                            dct:contributor [   sdo:name "Mattia Fumagalli"@en ;
+                                                sdo:identifier <https://orcid.org/0000-0003-3385-4769> ;
+                                                sdo:affiliation <https://www.inf.unibz.it/krdb/> ; ] .
+
+
+<https://www.inf.unibz.it/krdb/>  rdf:type sdo:Organization ;
```
Comment (tgoprince) [inline ontouml.ttl]: We use foaf instead of sdo. Shouldn't we use `foaf:Organization` instead?

File: ontouml.ttl [hunk 5]
```diff
@@ -1,303 +1,800 @@
...
+                                  sdo:name "KRDB Research Centre for Knowledge and Data, Free University of Bozen-Bolzano, Italy" ;
```
Comment (tgoprince) [inline ontouml.ttl]: Same thing here. `sdo:name` => `foaf:name`

File: ontouml.ttl [hunk 6]
```diff
@@ -1,303 +1,800 @@
...
+                                        [ sdo:name "Claudenir M. Fonseca"@en ;
+                                          sdo:identifier <https://orcid.org/0000-0003-2528-3118>;
+                                          sdo:affiliation <https://www.utwente.nl/en/eemcs/scs/> ; ] ,
+                                        [ sdo:name "Pedro Paulo Favato Barcelos"@en ;
+                                          sdo:identifier <https://orcid.org/0000-0003-2736-7817>;
+                                          sdo:affiliation <https://www.utwente.nl/en/eemcs/scs/> ; ] ;
+
+                            dct:contributor [   sdo:name "Mattia Fumagalli"@en ;
+                                                sdo:identifier <https://orcid.org/0000-0003-3385-4769> ;
+                                                sdo:affiliation <https://www.inf.unibz.it/krdb/> ; ] .
```
Comment (tgoprince) [inline ontouml.ttl]: I am not sure what the `foaf` alternative for `sdo:affiliation` is.

File: ontouml.ttl [hunk 7]
```diff
@@ -1,303 +1,800 @@
...
-                     rdfs:range ontouml:ElementView .
 
-ontouml:diagram a owl:ObjectProperty ;
+###  https://w3id.org/ontouml#categorizer
+ontouml:categorizer rdf:type owl:ObjectProperty ;
+                    rdfs:domain ontouml:GeneralizationSet ;
+                    rdfs:range ontouml:Class ;
+                    rdfs:label "categorizer"@en ;
+                    rdfs:isDefinedBy <https://w3id.org/ontouml> ;
+                    rdfs:comment "Identifies individuals of an ontouml:GeneralizationSet that are instances of a higher order ontouml:Class."@en ;
```
Comment (tgoprince) [inline ontouml.ttl]: This definition is wrong.

File: ontouml.ttl [hunk 8]
```diff
@@ -1,303 +1,800 @@
...
+                            dcat:landingPage <https://w3id.org/ontouml/vocabulary> ;
+                            vann:preferredNamespacePrefix "ontouml" ;
+                            vann:preferredNamespaceUri "https://w3id.org/ontouml#"^^xsd:anyURI ;
+                            foaf:homepage <https://purl.org/> ;
+                            dct:language <http://id.loc.gov/vocabulary/iso639-1/en> ;
+                            dct:isReferencedBy <https://dblp.org/rec/conf/jowo/FonsecaSVFGA21.html> ,
+                                               <https://doi.org/10.1007/978-3-031-17995-2_1> ;
+
+                            dcat:keyword "OntoUML"@en ,
+                                         "OntoUML-Metamodel"@en ,
```
Comment (tgoprince) [inline ontouml.ttl:44]: `"Metamodel"@en` suffices

File: ontouml.ttl [hunk 9]
```diff
@@ -1,303 +1,800 @@
...
+                                         "OntoUML-Vocabulary"@en ,
```
Comment (tgoprince) [inline ontouml.ttl]: This keyword is the name of the vocabulary.

File: docs/index-en.html.bak [hunk 10]
```diff
@@ -0,0 +1,4519 @@
+<!DOCTYPE html>
```
Comment (tgoprince) [inline docs/index-en.html.bak]: Why do we have a backup file in this repository?

File: docs/provenance/provenance-en.ttl [hunk 11]
```diff
@@ -0,0 +1,19 @@
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix : <> .
+<https://w3id.org/ontouml/1.1.0> a prov:Entity;
+	 dc:title "OntoUML Metamodel Vocabulary";
```
Comment (tgoprince) [inline docs/provenance/provenance-en.ttl]: The title here also needs to be changed.

File: docs/index-en.html [hunk 12]
```diff
@@ -0,0 +1,4245 @@
...
+          <li> <a href="#typeNature" title="https://w3id.org/ontouml#typeNature">typeNature</a>
+          </li>
+        </ul>
+      </div>
+      <!--DESCRIPTION SECTION-->
+      <div id="description">
+        <h2 id="desc" class="list">OntoUML Metamodel Vocabulary: Description <span
+
+            class="backlink"> back to <a href="#toc">ToC</a></span></h2>
+        <span class="markdown"> This is a placeholder text for the description
```
Comment (tgoprince) [inline docs/index-en.html]: You forgot to add some text here.

File: docs/index-en.html [hunk 13]
```diff
@@ -0,0 +1,4245 @@
...
+          of your ontology. The description should include an explanation and a
+          diagram explaining how the classes are related, examples of usage,
```
Comment (tgoprince) [inline docs/index-en.html]: Adding a diagram is a great idea to support documentation. I find it very helpful in DCAT and Ontolex.

File: docs/index-en.html [hunk 14]
```diff
@@ -0,0 +1,4245 @@
...
+    loadHash();
+}); $.fn.ignore = function(sel){
+        return this.clone().find(sel||">*").remove().end();
+ }; 
+   </script>
+  </head>
+  <body>
+    <div class="container">
+      <div class="head">
+        <div style="float:right">language <a href="index-en.html"><b>en</b></a>
```
Comment (tgoprince) [inline docs/index-en.html]: Since we only have the document in English, could we remove this option?

File: docs/index-en.html [hunk 15]
```diff
@@ -0,0 +1,4245 @@
...
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+      <!--OVERVIEW SECTION-->
+      <div id="overview">
+        <h2 id="overv" class="list">OntoUML Metamodel Vocabulary: Overview <span
+
+            class="backlink"> back to <a href="#toc">ToC</a></span></h2>
```
Comment (tgoprince) [inline docs/index-en.html]: Could we remove this links? They make the document cluttered.

File: docs/index-en.html [hunk 16]
```diff
@@ -0,0 +1,4245 @@
...
+        <span class="markdown"> This is a placeholder text for the description
+          of your ontology. The description should include an explanation and a
+          diagram explaining how the classes are related, examples of usage,
+          etc.</span> </div>
+      <!--CROSSREF SECTION-->
+      <div id="crossref">
+        <h2 id="crossreference" class="list">Cross-reference for OntoUML
+          Metamodel Vocabulary classes, object properties and data properties <span
```
Comment (tgoprince) [inline docs/index-en.html]: Could we change this title? It is a bit cryptic.

File: docs/index-en.html [hunk 17]
```diff
@@ -0,0 +1,4245 @@
...
+          <li> <a href="#triggers" title="https://w3id.org/ontouml#triggers">triggers</a>
+          </li>
+          <li> <a href="#type" title="https://w3id.org/ontouml#type">type</a> </li>
+          <li> <a href="#typeNature" title="https://w3id.org/ontouml#typeNature">typeNature</a>
+          </li>
+        </ul>
+      </div>
+      <!--DESCRIPTION SECTION-->
+      <div id="description">
+        <h2 id="desc" class="list">OntoUML Metamodel Vocabulary: Description <span
```
Comment (tgoprince) [inline docs/index-en.html]: This could be replaced by a Usage Guide section.

File: docs/index-en.html [hunk 18]
```diff
@@ -0,0 +1,4245 @@
...
+ }; 
+   </script>
+  </head>
+  <body>
+    <div class="container">
+      <div class="head">
+        <div style="float:right">language <a href="index-en.html"><b>en</b></a>
+        </div>
+        <h1>OntoUML Metamodel Vocabulary</h1>
+        <h2>Release 2023-07-23</h2>
```
Comment (tgoprince) [inline docs/index-en.html]: We should either use semantic versioning or date-based versioning.

**Review by tgoprince — CHANGES_REQUESTED** [review]

File: docs/index-en.html [hunk 19]
```diff
@@ -0,0 +1,4245 @@
...
+          <li> <a href="#upperBound" title="https://w3id.org/ontouml#upperBound">upperBound</a>
+          </li>
+          <li> <a href="#width" title="https://w3id.org/ontouml#width">width</a>
+          </li>
+          <li> <a href="#xCoordinate" title="https://w3id.org/ontouml#xCoordinate">xCoordinate</a>
+          </li>
+          <li> <a href="#yCoordinate" title="https://w3id.org/ontouml#yCoordinate">yCoordinate</a>
+          </li>
+        </ul>
+        <h4>Annotation Properties</h4>
```
Comment (tgoprince) [inline docs/index-en.html]: We should remove this section. These are just the properties we use to specify our metadata. They are not part of our specification.

File: docs/index-en.html [hunk 20]
```diff
@@ -0,0 +1,4245 @@
...
+                alt="https://creativecommons.org/licenses/by-sa/4.0"></a> </dd>
+          <!-- <dt>Evaluation:</dt><dd><a href="OOPSevaluation/oopsEval.html#" target="_blank"><img src="https://img.shields.io/badge/Evaluate_with-OOPS! (OntOlogy Pitfall Scanner!)-blue.svg" alt="Evaluate with OOPS!" /></a></dd> --><dt>Cite
+            as:</dt>
+          <dd>Sales, T. P., Fonseca, C. M., Barcelos, P. P. F. (2022) OntoUML Vocabulary Specification - Version 1.1.0. <a href="https://w3id.org/ontouml/1.1.0">https://w3id.org/ontouml/1.1.0</dd>
+        </dl>
+        <a href="provenance/provenance-en.html" target="_blank">Provenance of
+          this page</a>
+        <hr> </div>
+      <div class="status">
+        <div> <span>OntoUML Vocabulary Specification - Version 1.1.0</span> </div>
```
Comment (tgoprince) [inline docs/index-en.html]: I think we should remove this floating title on the top-left corner of the page.

**Review by tgoprince — CHANGES_REQUESTED** [review]

File: ontouml.ttl (hunk 3 above)
Comment (pedropaulofb) [reply ontouml.ttl]: Done.

File: ontouml.ttl (hunk 4 above)
Comment (pedropaulofb) [reply ontouml.ttl]: Done.

File: ontouml.ttl (hunk 5 above)
Comment (pedropaulofb) [reply ontouml.ttl]: Done.

File: ontouml.ttl (hunk 6 above)
Comment (pedropaulofb) [reply ontouml.ttl]: Done.

File: ontouml.ttl (hunk 8 above)
Comment (pedropaulofb) [reply ontouml.ttl:44]: The intention here is to make a direct reference to the OntoUML-Metamodel. In a (semantic or conventional) web application, a user may be interested in finding all resources build upon the OntoUML-Metamodel. E.g., a search engine will present better results when the searched resource makes a clear reference to the searched term.

@tgoprince and @claudenirmf, please verify the position to be adopted. If both of you agree that `"Metamodel"@en` is the best option, please do the modification. I will not oppose to that.

File: ontouml.ttl (hunk 9 above)
Comment (pedropaulofb) [reply ontouml.ttl]: Done.

File: README.md (hunk 1 above)
Comment (pedropaulofb) [reply README.md:4]: Done.

File: docs/provenance/provenance-en.ttl (hunk 11 above)
Comment (pedropaulofb) [reply docs/provenance/provenance-en.ttl]: Done.

File: ontouml.ttl (hunk 5 above)
Comment (pedropaulofb) [reply ontouml.ttl]: Just to be clearer:

1) According to FOAF itself:

"FOAF does not currently have a term for the name of the relation (eg. "workplace") that holds between a [foaf:Person](http://xmlns.com/foaf/0.1/#term_Person) and an [foaf:Organization](http://xmlns.com/foaf/0.1/#term_Organization) that they work for."

2) The default solution adopted by [Widoco is to use the sdo vocabulary](https://github.com/dgarijo/Widoco/issues/485).

3) As we do not want to use the sdo vocabulary, I registered the information in a correct way (organization foaf:member person)

4) The registered information is not read by widoco, so the information is manually inserted in Widoco at the html generation moment

File: docs/index-en.html (hunk 20 above)
Comment (pedropaulofb) [reply docs/index-en.html]: This is part of the W3C template. Its recommendations use this floating title. IMO, we should keep to be aligned with them.

However, @tgoprince and @claudenirmf, if you two decide to remove, I won't oppose to that.

File: docs/index-en.html (hunk 19 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done.

File: docs/index-en.html (hunk 18 above)
Comment (pedropaulofb) [reply docs/index-en.html]: This information is generated after the value we attributed to "dct:issued" in our metadata. We are using only semantic versioning. Note that this is part of the W3C template (their Recommendations always provide the release date on top). IMO, we should keep as it is. But, @tgoprince and @claudenirmf, if you decide to remove, I won't oppose.

File: docs/index-en.html (hunk 13 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done.

File: docs/index-en.html (hunk 12 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done.

File: docs/index-en.html.bak (hunk 10 above)
Comment (pedropaulofb) [reply docs/index-en.html.bak]: It was automatically generated by widoco. I'll remove it.

File: ontouml.ttl (hunk 7 above)
Comment (pedropaulofb) [reply ontouml.ttl]: Done.

File: docs/index-en.html (hunk 16 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done.

File: docs/index-en.html (hunk 17 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done.

File: docs/index-en.html (hunk 14 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done.

File: docs/index-en.html (hunk 15 above)
Comment (pedropaulofb) [reply docs/index-en.html]: IMHO, they are important for the document navigation, especially considering the large amount of entities' declarations that we have. I vote to leave it as it is. It is working properly now that I added a table of contents.

If @tgoprince and @claudenirmf decide to remove, I won't oppose.

**Comment by pedropaulofb** [comment]
@tgoprince and @claudenirmf , the PR is ready for a second review. 

This time we should not have link problems after creating the page.

File: docs/ontology.ttl [hunk 21]
```diff
@@ -0,0 +1,1680 @@
...
+@base <https://w3id.org/ontouml#> .
+
+<https://w3id.org/ontouml> rdf:type owl:Ontology ;
+                            owl:versionIRI <https://w3id.org/ontouml/1.1.0> ;
+                            <http://purl.org/dc/terms/conformsTo> <https://w3id.org/ontouml/metamodel/v1.1.0> ;
+                            <http://purl.org/dc/terms/contributor> _:genid1 ;
+                            <http://purl.org/dc/terms/creator> _:genid2 ,
+                                                               _:genid3 ,
+                                                               _:genid4 ;
+                            <http://purl.org/dc/terms/description> "A reference implementation of the OntoUML Metamodel in OWL."@en ;
```
Comment (tgoprince) [inline docs/ontology.ttl]: Could you use the `dct` prefix to simplify this file?

File: docs/index-en.html (hunk 18 above)
Comment (tgoprince) [reply docs/index-en.html]: To me, `Release 2023-07-23` suggests that this is the version of the documentation (or the vocabulary). In any case, I don' consider the issue date to be so important as to deserve being the subtitle of the document. This is just provenance data (which is available in a following link).

File: docs/index-en.html (hunk 16 above)
Comment (tgoprince) [reply docs/index-en.html]: The problem was with the "cross-reference" part. Why not call it simply "Vocabulary specification" as in [DCAT](https://www.w3.org/TR/vocab-dcat-2/#vocabulary-specification)?

File: docs/index-en.html (hunk 20 above)
Comment (tgoprince) [reply docs/index-en.html]: Why would we want to stick we the W3C template? We are not associated with them.

File: docs/index-en.html (hunk 15 above)
Comment (tgoprince) [reply docs/index-en.html]: I understand the function, but there are too many of them!

File: docs/index-en.html (hunk 13 above)
Comment (tgoprince) [reply docs/index-en.html]: Could we replace the abstract syntax diagram with the one I created for the paper? I can create another one in the same style for the concrete syntax part.

File: ontouml.ttl (hunk 5 above)
Comment (tgoprince) [reply ontouml.ttl]: This last comments regards the comment on `sdo:affiliation`, right?

File: ontouml.ttl (hunk 8 above)
Comment (tgoprince) [reply ontouml.ttl:44]: Got it. Then it seems that we could have both `metamodel` and `OntoUML Metamodel`

File: docs/index-en.html (hunk 13 above)
Comment (tgoprince) [reply docs/index-en.html]: metamodel-summary

**Review by tgoprince — CHANGES_REQUESTED** [review]

File: ontouml.ttl (hunk 5 above)
Comment (pedropaulofb) [reply ontouml.ttl]: It goes straight to the HTML information. It is not stored into the original ttl vocab file, so we don't have to worry about that.

File: docs/index-en.html (hunk 15 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Ok. Removed.

Comment (pedropaulofb) [reply docs/index-en.html]: Done. Removed.

File: docs/index-en.html (hunk 16 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done.

File: docs/index-en.html (hunk 18 above)
Comment (pedropaulofb) [reply docs/index-en.html]: I substituted the date by the version and added the modification date together with the other metadata

File: docs/index-en.html (hunk 20 above)
Comment (pedropaulofb) [reply docs/index-en.html]: Done

File: docs/ontology.ttl (hunk 21 above)
Comment (pedropaulofb) [reply docs/ontology.ttl]: The OWL, JSON-LD, and TTL files inside the `docs` folder is generated automatically by Widoco after our original ttl file. I believe that the intention is to allow the vocabulary to be used by a larger number of computational tools.

Regarding the TTL file, I just adjusted the link to download the file directly from the original source (it makes much more sense, right!? hehe)
Regarding the other formats, I let them as they are now.

**Comment by pedropaulofb** [comment]
I addressed all comments. Tiago, please reevaluate.

**Review by tgoprince — APPROVED** [review]

**Pull request merged**